CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -2987,22 +3145,22 @@ int SiPixelTemplate::qbin(int id,
 
```

Comment:
```ilow``` and ```ihigh``` as assigned in previous lines 3142 and 3143 are never used, and that assignment can be removed (see static analyzer output)

Same for the assignment of the same two variables at line 3631 and 3815

Reply:
Ok I removed them